extension-types.md

```diff
@@ -49,18 +49,16 @@ UTF-8-encoded string.
 
 The following metadata keys are supported:
 
-- `crs_encoding`: Contains a UTF-8-encoded string describing how
-  the `crs` metadata field should be interpreted. Supported values of
-  `crs_encoding` are `"projjson"` and `"wkt2"`. If possible, producers
-  should encode this value as `"projjson"`. If this key is omitted
-  the consumer cannot assume any specific encoding (i.e., producers
-  can pass on CRS information in whatever form they received it). This
-  metadata should not be present if `crs` is not present.
-- `crs`: Contains a serialized version of the coordinate reference system.
+- `crs`: Contains a serialized version of the coordinate reference system
```

Comment:
I don't think this is settled per the GitHub issue.  It should be wkt, wkid, wkt2, or projjson

Reply:
I would say WKT/WKT2 is more well known then PROJJSON.  If we settle on projjson then right out the bat many programs can't use it.

Reply:
Let's get it right now, so people who adopt draft 1 won't have to go back and adjust their logic.  There is nothing wrong with accepting the following formats.  This just makes PROJJSON a defacto primary format when most products don't use that format.  I don't see what is wrong with stating the project accepts a wide variety of spatial reference formats: WKT, WKID, WKT2, and PROJJSON formats, know this the metadata specifies the format and it is the responsibility of the consumer to be able to understand the output spatial reference system.

Reply:
@achapkowski I know this discussion is not yet settled, but I don't want to further block merging this PR and starting with _something_ so we can start trying this out in practice. I personally still think it would be good to see if we can relax this in some way, see also my comment at https://github.com/geopandas/geo-arrow-spec/issues/8#issuecomment-1164285103

But so I am going to merge this PR now, we can continue the discussion in https://github.com/geopandas/geo-arrow-spec/issues/8